packages/page-experience/lib/checks/fontDisplay.js

```diff
@@ -30,6 +45,11 @@ const checkDisplayOptional = (pageData, result) => {
     }
     const isGoogleFont =
       fontface.mainSrc && fontface.mainSrc.startsWith('https://fonts.gstatic.com');
+    const isIconFont = ICON_FONTS.has(fontface.fontFamily);
```

Comment:
This will not catch the aforementioned font family ```Font Awesome 5``` unless it is added to the set. Another way would be to check if a substring of the font family is found in the array like:
```
const isIconFont = ICON_FONTS.some((font) => {
  return fontface['fontFamily'].includes(font)
});
```

Reply:
good tip! added!